app/views/incomes/YourIncomeCalculationView.scala.html

```diff
@@ -32,155 +32,146 @@
     ec: scala.concurrent.ExecutionContext)
 
 
-@printLink= {
```

Comment:
Just double checking, the new print button now routes to the old view temporarily?

Reply:
yes it does